src/isomorphic/classic/types/checkTypes.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule checkTypes
```

Comment:
Can we call this checkPropTypes or something else more descriptive? (I know you don't really like calling the context types prop types.) This name needs to be unique across the whole Facebook codebase.